libs/media-player/src/lib/IcecastStreamPlayer.tsx

```diff
@@ -0,0 +1,17 @@
+interface IcecastPlayerProps {
+    audioUrl: string;
+    listenMessage?: string;
+}
+
+const DEFAULT_LISTEN_MESSAGE = 'Listen Live!';
+
+export const IcecastStreamPlayer = ({ audioUrl, listenMessage }: IcecastPlayerProps) => {
+    return (
+        <div>
+            <audio controls>
+                <source src={audioUrl} type="audio/ogg" />
```

Comment:
This makes me a little nervous. I suppose you're just providing a fallback list.